### DISCUSSION TITLE
Support for Kubernetes 1.29

### DISCUSSION THREAD

**Discussion by Sonoko-Ishiki** [prometheus-operator/kube-prometheus#2373] [Q&A]
The README.md states that the current main branch is compatible with Kubernetes 1.28, but will it also work with 1.29?

**Comment by ameg** [comment]
I would also greatly appreciate any information or experience regarding 1.29/1.30 support, as 1.28 EOL is coming up soon:

 - Kubernetes 1.28 active support ends: 28 August 2024
 - Kubernetes 1.28 maintenance support ends: 28 October 2024
 - EKS standard support ends: 26 November 2024
 - GKE standard support ends: 31 December 2024

**Reply by ashish1099** [reply]
so far v0.13.0 works fine on k8s 1.29

**Reply by ashish1099** [reply]
but yes a stable release would be fine for 1.29/1.30 k8s
